From f64a023abfb6c645fbf7797d17bfae50d8b325bc Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:30:24 +0800
Subject: [PATCH] configs: evb-ast2705: switch to CONFIG_ASPEED_AST2705

Replace CONFIG_ASPEED_AST2700 with CONFIG_ASPEED_AST2705 now that
the dedicated AST2705 SoC config symbol exists.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: I12235707f3447af8f140e9016d7f46f5276d408b
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 configs/evb-ast2705_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/evb-ast2705_defconfig b/configs/evb-ast2705_defconfig
index f9f574131f..a1ada5a91d 100644
--- a/configs/evb-ast2705_defconfig
+++ b/configs/evb-ast2705_defconfig
@@ -6,7 +6,7 @@ CONFIG_ARM_SMCCC=y
 CONFIG_ARCH_ASPEED=y
 CONFIG_TEXT_BASE=0x400000000
 CONFIG_SYS_MALLOC_LEN=0x2000000
-CONFIG_ASPEED_AST2700=y
+CONFIG_ASPEED_AST2705=y
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0x403000000
-- 
2.43.0

